### PR TITLE
Tank rogue implementation

### PR DESCRIPTION
**Pull request by catszeid**
Implement tanking for Rogue. Preset gear, talents, and rotations.

### CODE REVIEW

File: sim/rogue/runes.go [hunk 1]
```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
 		rogue.SliceAndDiceAura.Activate(sim)
 	}
 }
+
+// TODO: Update cut to the chase to refresh
```
Comment (kayla-glick) [inline sim/rogue/runes.go:161]: Looks like the duration refresh is still missing here
https://github.com/wowsims/sod/blob/79f61bb7d38978f4d6d81e1ae94e24cf3fa6f58d/sim/rogue/runes.go#L152-L159

File: sim/rogue/runes.go [hunk 2]
```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
...
+	// TODO: Confirm threat mod
+	rogue.PseudoStats.ThreatMultiplier *= 1.895
+
+	// Blade Dance 20% Physical DR - Added in registerBladeDance()
+
+	// -6% to be critically hit
+	rogue.PseudoStats.ReducedCritTakenChance += 6
+
+	// Replace Feint with Tease
+	// Warrior sim from wrath did not implement it. May implement later
```
Comment (kayla-glick) [inline sim/rogue/runes.go:240]: Could you add a TODO on this comment if you're not planning to include it for now?

File: sim/rogue/runes.go [hunk 3]
```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
...
+	})
+}
+
+func (rogue *Rogue) applyJustAFleshWound() {
+	if !rogue.HasRune(proto.RogueRune_RuneJustAFleshWound) {
+		return
+	}
+	// Mod threat
+	// TODO: Confirm threat mod
+	rogue.PseudoStats.ThreatMultiplier *= 1.895
```
Comment (kayla-glick) [inline sim/rogue/runes.go:232]: Any updates on this? If not not the biggest deal

File: sim/rogue/runes.go [hunk 4]
```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
...
+	rogue.PseudoStats.ReducedCritTakenChance += 6
+
+	// Replace Feint with Tease
+	// Warrior sim from wrath did not implement it. May implement later
+
+	// Shuriken Toss and Poisoned Knife gain 50% threat mod
+	// Implemented in the relevant files
+}
+
+func (rogue *Rogue) applyRollingWithThePunches() {
```
Comment (kayla-glick) [inline sim/rogue/runes.go:246]: Looks like this is still missing
> In addition, each time you Dodge or Parry, you gain 4 Attack Power for each point of your defense skill beyond [50 * level]. Lasts 15 sec.

File: sim/rogue/runes.go (hunk 3 above)
Comment (catszeid) [reply sim/rogue/runes.go:245]: Not right now. Need to do in-game testing to confirm it.

File: sim/rogue/runes.go (hunk 4 above)
Comment (catszeid) [reply sim/rogue/runes.go:259]: I will add this when I do the rest of the phase 4 runes. Unsure if the implementation will change between what they have now and the next update for PTR

**Comment by kayla-glick** [comment]
@catszeid let me know when you'd like another review or if it's good to merge!

**Pull request merged**